code/modules/events/fugitive_spawning.dm

```diff
@@ -133,7 +133,8 @@ GLOBAL_LIST_EMPTY(fugitive_backstory_selection)
 
 /proc/admin_select_backstory(list/backstory_keys)
 	GLOB.fugitive_backstory_selection = backstory_keys
-	message_admins("Choosing random fugitive backstory in 15 seconds. \
+	message_admins("Choosing random fugitive backstory in 30 seconds. \
 		<a href='?_src_=holder;[HrefToken(TRUE)];backstory_select=[REF(backstory_keys)]'>SELECT MANUALLY</a>")
-	sleep(15 SECONDS)
+	play_sound_to_all_admins('sound/effects/admin_alert.ogg')
+	sleep(30 SECONDS)
```

Comment:
30 is way too long for this since it also triggers on top of the regular fugitives delay. make it like 20 maximum

Reply:
Done